api/cmd/portainer/main.go

```diff
@@ -100,6 +101,14 @@ func retrieveFirstEndpointFromDatabase(endpointService portainer.EndpointService
 	return &endpoints[0]
 }
 
+func retrievePassword(flags *portainer.CLIFlags) string {
+  password := *flags.AdminPassword
+  if !strings.HasPrefix(password, "admin:") {
```

Comment:
I think that the password format should be without the `admin:` (this flag should not be locked on `htpasswd` format).

A user will be able to generate a password using `httpd htpasswd -nb -B admin password | cut -d ':' -f 2`.